gtk4/sys/src/lib.rs

```diff
@@ -802,6 +802,7 @@ pub const GTK_ACCESSIBLE_VALUE_UNDEFINED: c_int = -1;
 pub const GTK_IM_MODULE_EXTENSION_POINT_NAME: *const c_char =
     b"gtk-im-module\0" as *const u8 as *const c_char;
 pub const GTK_INPUT_ERROR: c_int = -1;
+pub const GTK_INVALID_LIST_POSITION: c_int = 4294967295;
```

Comment:
We should modify the one we use in gtk4/src/lib.rs to use the ffi value

Reply:
ok, missed that

Reply:
Had to fix GI: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/3738